pkg/prometheus/server/statefulset_test.go

```diff
@@ -2353,14 +2351,17 @@ func TestPrometheusAdditionalArgsNoError(t *testing.T) {
 		"--storage.tsdb.no-lockfile",
 	}
 
+	expectedPrometheusArgsV2 := append([]string{"--web.console.templates=/etc/prometheus/consoles",
```

Comment:
I wouldn't mind code duplication in tests. If you want, you can also use table-driven tests with 1 input for v2 and 1 input for v3.

https://go.dev/wiki/TableDrivenTests

Reply:
Agreed, refactored so the test is using table driven test.